### DISCUSSION TITLE
SentenceSegmenter in v3?

### DISCUSSION THREAD

**Discussion by shahidkhan021** [explosion/spaCy#9198] [Help: Other Questions] [answered]
in sapcy 2.0 i was using from spacy.pipeline import SentenceSegmenter what is equivalent in spacy 3

**Comment by polm** [comment, marked as answer]
See the [Sentencizer](https://spacy.io/api/sentencizer). 

Note that even in the later releases of v2 the SentenceSegmenter was deprecated in favor of the Sentencizer.

**Reply by shahidkhan021** [reply]
can i do custom sentence splitting

**Reply by svlandeg** [reply]
Sure. There is plenty of documentation around this here: https://spacy.io/usage/linguistic-features#sbd
